docs/pyproject.toml

```
@@ -9,7 +9,7 @@ authors = ["ScyllaDB Documentation Contributors"]
 python = "^3.9"
 pyyaml = "6.0.1"
 pygments = "2.15.1"
-sphinx-scylladb-theme = "~1.6.6"
+sphinx-scylladb-theme = "^1.7.1"
```

```suggestion
sphinx-scylladb-theme = "~1.7.1"
```
`minor` upgrades need manual changes

---

done